### PR TITLE
Feature/#2140 package install

### PR DESCRIPTION
**Pull request by gabrielbarker**
### What does this PR do?
This PR adds a command to install a package into the default org.

### What issues does this PR fix or reference?
#2140 

### Functionality Before
This had to be done through the org or CLI

### Functionality After
There is a command, SFDX: Install Package, to the vscode command palette.

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/commands/forcePackageInstall.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
```
Comment (sfsholden) [inline packages/salesforcedx-vscode-core/src/commands/forcePackageInstall.ts:2]:
```suggestion
 * Copyright (c) 2020, salesforce.com, inc.
```

File: packages/salesforcedx-vscode-core/src/commands/forcePackageInstall.ts [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+  public execute(
+    response: ContinueResponse<PackageIdAndInstallationKey>
+  ): void {
+    const startTime = process.hrtime();
+    const cancellationTokenSource = new vscode.CancellationTokenSource();
+    const cancellationToken = cancellationTokenSource.token;
+
+    const execution = new CliCommandExecutor(this.build(response.data), {
+      cwd: getRootWorkspacePath()
```
Comment (sfsholden) [inline packages/salesforcedx-vscode-core/src/commands/forcePackageInstall.ts:69]: We may want to add this piece as well...
env: { SFDX_JSON_TO_STDOUT: 'true' }

Comment (gabrielbarker) [reply packages/salesforcedx-vscode-core/src/commands/forcePackageInstall.ts]: Done! Thanks for pointing this out

**Comment by lcampos** [comment]
I'm closing this since these changes got merged as part of PR #2244. @gabrielbarker thanks again for your contribution, the changes will go out as part of this week's release `v48.20.0`

**Comment by gabrielbarker** [comment]
Thanks @lcampos and team! I'm glad I could help

**Pull request closed without merging**